modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-0_{context}"]
+= Release notes for {gitops-title} 1.7.0
+
+{gitops-title} 1.7.0 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="new-features-1-7-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, you can add environment variables to the Notifications controller. link:https://issues.redhat.com/browse/GITOPS-2313[GITOPS-2313]
+
+* With this update, the default nodeSelector `"kubernetes.io/os": "linux"` key-value pair is added to all workloads such that they only schedule on Linux nodes. In addition, any custom node selectors are added to the default and take precedence if they have the same key. link:https://issues.redhat.com/browse/GITOPS-2215[GITOPS-2215]
+
+* With this update, you can set custom node selectors in the Operator workloads by editing their `GitopsService` custom resource. link:https://issues.redhat.com/browse/GITOPS-2164[GITOPS-2164]
+
+
+* With this update, you can use the RBAC policy matcher mode to select from the following options: `glob` (default) and `regex`.link:https://issues.redhat.com/browse/GITOPS-1975[GITOPS-1975]
+
+* With this update, you can customize resource behavior using the following additional subkeys:
+
+|===
+| Subkey | Key form | Mapped field in argocd-cm
+| resourceHealthChecks | resource.customizations.health.<group_kind> | resource.customizations.health
+| resourceIgnoreDifferences | resource.customizations.ignoreDifferences.<group_kind> | resource.customizations.ignoreDifferences
+| resourceActions | resource.customizations.actions.<group_kind> | resource.customizations.actions
+|===
++
+link:https://issues.redhat.com/browse/GITOPS-1561[GITOPS-1561]
++
+[NOTE]
```

Comment:
There's a rendering issue in the preview where the `+` signs are displaying. Maybe adding the `+` in line 26 will fix it.